_posts/en/newsletters/2021-08-04-newsletter.md

```diff
@@ -63,7 +63,15 @@ guidelines for the addition of new tracepoints.
 
 - [Eclair #1893][] Different default relay fees for announced and unannounced channels FIXME:bitschmidty
 
-- [Rust-Lightning #967][] valentinewallace/2021-06-keysend FIXME:dongcarl
+- [Rust-Lightning #967][] adds support for making [keysend-style spontaneous
+  payments][topic spontaneous payments#add-data-to-the-routing-packet] via the
```

Comment:
Unfortunately markdown doesn't let you append a hash to a link descriptor like that.  You'll have to add a link definition pointing to `/en/topics/spontaneous-payments/#add-data-to-the-routing-packet` to the bottom of the page.